blocks/header/alt-0-nav.css

```diff
@@ -35,14 +39,21 @@
   }
 }
 
+/** apply this styling when dealing with alt-0 nav **/
```

Comment:
This is a kind of anti pattern that through header we are setting style for the footer

Reply:
@MarquiseRosier Can you please check this.